CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff

```diff
@@ -1756,7 +1756,7 @@ Sie enthält derzeit %2$lld pins.</target>
       </trans-unit>
       <trans-unit id="Thank you for trying out Callsheet. 💙" xml:space="preserve">
         <source>Thank you for trying out Callsheet. 💙</source>
-        <target state="translated">Danke, dass du Callsheet ausprobiert hast. 💙</target>
+        <target state="translated">Danke, dass du Callsheet ausprobierst. 💙</target>
```

Comment:
"ausprobiert hast" is past tense, which I don't think you want here. Changed it to present tense.